pkg/provider/onepassword/onepassword.go

```diff
@@ -260,11 +432,11 @@ func (provider *ProviderOnePassword) findItem(name string) (*onepassword.Item, e
 		case len(items) == 1:
 			return provider.client.GetItemByUUID(items[0].ID, items[0].Vault.ID)
 		case len(items) > 1:
-			return nil, fmt.Errorf(errExpectedOneItem, fmt.Errorf(incorrectCountFormat, name, len(items)))
+			return nil, fmt.Errorf("%w: '%s', got %d", ErrExpectedOneItem, name, len(items))
 		}
 	}
 
-	return nil, fmt.Errorf(errKeyNotFound, fmt.Errorf("%s in: %v", name, provider.vaults))
```

Comment:
Why the deletion of these error messages and replace them with the formatting?

Reply:
this is done to use the `%w` formatting verb, which will allow any downstream code to use the following to check types:
```go
error.Is(err, ErrKeyNotFound)
```

Reply:
Ah good point. I haven't noticed that.